content/en/account_management/billing/profiler.md

```diff
@@ -23,7 +23,7 @@ Using five hosts running one application being profiled in each host. No contain
 | Billable Unit  | Quantity   | Price                                                                                           | Formula       | Subtotal              |
 |----------------|------------|-------------------------------------------------------------------------------------------------|---------------|-----------------------|
 | APM Hosts      | 5          | $12 per host                                                                                    | 5 * $12       | $60                   |
-| Profiled containers  | 0 per host | $0 since there is no overage                                                              | 0 * $2        | $0                    |
+| Profiled containers  | 0 per host | $0 since there is no additional containers                                                              | 0 * $2        | $0                    |
```

Comment:
```suggestion
| Profiled containers  | 0 per host | $0 since there are no additional containers                                                              | 0 * $2        | $0                    |
```